prow/Makefile

```diff
@@ -12,25 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+include Makefile.gcloud.mk
```

Comment:
Since this isn't istio, can we use bazel rules and avoid Makefiles?

Reply:
Yes, we can do that. I created this #97. The Makefile is simple enough so that it shouldn't be too much effort to port.